src/ice_shelf/MOM_ice_shelf_dynamics.F90

```diff
@@ -741,8 +759,8 @@ subroutine ice_shelf_advect(CS, ISS, G, time_step, Time)
 !    into partial cells
 
   real, dimension(SZDI_(G),SZDJ_(G))   :: h_after_uflux, h_after_vflux ! Ice thicknesses [Z ~> m].
-  real, dimension(SZDIB_(G),SZDJ_(G))  :: uh_ice  ! The accumulated zonal ice volume flux [Z L2 ~> m3]
-  real, dimension(SZDI_(G),SZDJB_(G))  :: vh_ice  ! The accumulated meridional ice volume flux [Z L2 ~> m3]
+  real, dimension(SZDI_(G),SZDJ_(G))  :: uh_ice  ! The accumulated zonal ice volume flux [Z L2 ~> m3]
```

Comment:
These declarations were already appropriate for variables on a C-grid, and should not have been changed.